web/cypress/integration/Home/EditTest.spec.ts

```diff
@@ -1,43 +1,43 @@
-import {createTest, deleteTest, testId} from '../utils/Common';
+import {createTest, testRunPageRegex} from '../utils/Common';
 
 describe('Edit Test', () => {
-  before(() => {
-    createTest();
-  });
-
-  after(() => {
-    deleteTest();
-  });
-
-  beforeEach(() => {
-    cy.visit('http://localhost:3000/');
-  });
-
   it('should edit a test', () => {
-    cy.get(`[data-cy=test-actions-button-${testId}]`).click();
-    cy.get('[data-cy=test-card-edit]').click();
+    (async () => {
```

Comment:
this is the fix of our issues. 
some-why tests were being marked as "passing" before the last promise returns.
those was the weird behaviour we were seeing